waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/service/impl/PrefixBasedDatabaseMappingService.java

```diff
@@ -239,74 +247,95 @@ public DatabaseMapping databaseMapping(@NotNull String databaseName) {
     return mappings;
   }
 
-  private boolean isWhitelisted(String databasePrefix, String database) {
-    Whitelist whitelist = mappedDbByPrefix.get(databasePrefix);
-    if ((whitelist == null) || whitelist.isEmpty()) {
-      // Accept everything
-      return true;
-    }
-    return whitelist.contains(database);
-  }
+//  private boolean isWhitelisted(String databasePrefix, String database) {
+//    Whitelist whitelist = mappedDbByPrefix.get(databasePrefix);
+//    if ((whitelist == null) || whitelist.isEmpty()) {
+//      // Accept everything
+//      return true;
+//    }
+//    return whitelist.contains(database);
+//  }
 
   @Override
   public PanopticOperationHandler getPanopticOperationHandler() {
     return new PanopticOperationHandler() {
 
+      private void sleepThread() {
+        try {
+          Thread.sleep(2000L);
+        } catch (InterruptedException e) {
+          LOG.info("Execution was cancelled", e.getMessage());
```

Comment:
cancelled -> interrupted
Might as well log `e` (this should never happen so having the whole stack could be useful)